### ISSUE TITLE
Reverse of entropy

### ISSUE THREAD

**Issue by benam2** [JohnTailor/tkm#14]
Hello,

Actually, I found your project very interesting. I have a confusion about the reverse of entropy idea.
My question is more about the implementation part:
suppose we have a matrix (10*2) stands for 10 words and 2 topics. 
From your code, I can understand that you have applied the entropy on each row, so in this case, we will end up with 10 entropy. then how you can decide about the distinctiveness of the clusters of the topics?
To put it another way, how can you derive a conclusion which word is distinctive to the two available topics?

Thank you

**Comment by JohnTailor** [comment]
given you have a 2x3 word topic distribution
             topic1  topic2  topic3
word1     10         10        10
word2      5            0          0


word1 is not indicative of any topic, word2 for topic1
Entropy
Normalize:
              topic1  topic2  topic3
word1     .33         .33        .33
word2      1            0          0


Entropy word1 = log 0.33 = -.48
Entropy word2 = log 1 = 0

You could also use other measures for concentration then entropy.

Hope this helps.
